pythonx/UltiSnips/snippet/definition/_base.py

```diff
@@ -43,7 +44,7 @@ class SnippetDefinition(object):
     _TABS = re.compile(r"^\t*")
 
     def __init__(self, priority, trigger, value, description,
-                 options, globals, location):
+                 options, globals, location, context=None):
```

Comment:
please do not use default values, instead specific the context as None everywhere.